src/main/webapp/css/upload.css

```diff
@@ -1,3 +1,7 @@
+body {
+  display: none;
+}
```

Comment:
This should be added to style.css since it's being applied to every page now. We may also want to create a JavaScript file shared by all pages to avoid repeated code.

Reply:
Shared CSS and JS have been added to #63.